From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 25 Jul 2019 18:34:08 -0300
Subject: Remove setcap

The debian setcap is applied in postinst instead (and the setcap here
doesn't work with debian's temporary install location).
---
 daemon/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/CMakeLists.txt b/daemon/CMakeLists.txt
index bf3e540..ee6121a 100644
--- a/daemon/CMakeLists.txt
+++ b/daemon/CMakeLists.txt
@@ -35,7 +35,7 @@ else()
         install(PROGRAMS ${CMAKE_SOURCE_DIR}/lokinet-bootstrap DESTINATION bin COMPONENT lokinet)
     endif()
 
-    if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+    if(FALSE AND ${CMAKE_SYSTEM_NAME} MATCHES "Linux")
         install(CODE "execute_process(COMMAND setcap cap_net_admin,cap_net_bind_service=+eip ${CMAKE_INSTALL_PREFIX}/bin/lokinet)")
     elseif(${CMAKE_SYSTEM_NAME} MATCHES "FreeBSD")
         target_link_directories(${EXE} PRIVATE /usr/local/lib)
